WireAnalytics/Sources/WireAnalytics/Events/Predefined/AnalyticsEvent+Calling.swift

```diff
@@ -72,5 +72,107 @@ public extension AnalyticsEvent {
             .init(name: "calling.call_quality_review", segmentation: review.segmentation)
         }
 
+        /// An event tracking when a call ends.
+
+        public static func endedCall(
+            deviceModel: String,
+            osVersion: String,
+            callEndReason: CallEndedReason,
+            callDetails: CallDetails,
+            conversationDetails: ConversationDetails,
+            isTeamMember: Bool
+        ) -> AnalyticsEvent {
+            AnalyticsEvent(name: "calling.ended_call", segmentation: [
+                .deviceModel(deviceModel),
+                .osVersion(osVersion),
```

Comment:
Isn't this base segmentation already included when we pass the event to Countly?

Reply:
True, thanks, `AnalyticsService` adds them, fixed: c81325151ab27d7a36acef0162a84cdf7cf83bcf